src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -743,13 +735,14 @@ class ActivityResourceIntTest : IntegrationTestBase() {
       @Test
       fun `should return OK if pay rate removed which is NO LONGER allocated to an offender`() {
         testData(repository) {
-          offender = offender(nomsId = "A1234TT") {
```

Comment:
Get rid of references to offender because it's only used to get the latest booking anyway